### DISCUSSION TITLE
YOLO Achievement not showing on profile.

### DISCUSSION THREAD

**Discussion by Ron-Caster** [community/community#61604] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

In my GitHub profile, the Y.O.L.O Achievement is showing in mobile version in my iPad's GitHub app. But it is not showing when I go to my profile via browser. Why?

**Comment by radar-blip** [comment, marked as answer]
I can see that you have it.
Try going here: https://github.com/Ron-Caster?tab=achievements
and unhide the badge by clicking the eye icon.

**Reply by Ron-Caster** [reply]
Thank you <3

**Reply by radar-blip** [reply]
Anytime. 👍🏻
